<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>50</y>
      <width>381</width>
      <height>191</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>C++ code...</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="variableTextEdit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>181</width>
      <height>191</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="functionTextEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>181</width>
      <height>191</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="openFileBtn">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>250</y>
      <width>381</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parseLVBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Parse local variables</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parseFOBtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Parse functions</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="countEdit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>340</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>320</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Structs, classes, arrays</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Variable changes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Functions</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parseVCBtn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>250</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Parse variable changes</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="localVariableTextEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>181</width>
      <height>191</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Local variables</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="allVariablesTextEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>181</width>
      <height>191</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>All variables</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parseVBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>520</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Parse variables</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="depthEdit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>400</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>380</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Branch depths</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parseMiscBtn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Parse</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="functionOverloadTextEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>320</y>
      <width>181</width>
      <height>191</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="parseFOBtn_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>520</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Parse functions</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Function overloads</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="logicErrorTextEdit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>320</y>
      <width>181</width>
      <height>191</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="parseLEBtn">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>520</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Parse functions</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1154</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
